tests/test_lambdas/test_broker.py

```diff
@@ -18,21 +19,21 @@ def func(_data: dict) -> LambdaResponse:
         mapper = {"x": func}
         event = {"op": "x", "data": {"y": 1}}
 
-        resp = LambdaBroker(mapper, event).react()
+        resp = LambdaBroker(mapper, event, LambdaContext()).react()
 
         assert resp == {
             "result": LambdaResult.OK,
             "data": {"some": "data"},
         }
 
     def test_broker_works_properly_when_no_data_is_provided(self) -> None:
```

Comment:
This whole test is no longer valid as we resigned from this concept - I would remove it 😉